org-timeline.el

```diff
@@ -62,6 +62,20 @@
   :type 'boolean
   :group 'org-timeline)
 
+(defcustom org-timeline-start-hour 5
+  "Starting hour of the timeline."
+  :type 'integer
+  :group 'org-timeline)
+
+(defconst org-timeline-first-line 0
+  "Computer first line of the timeline in the buffer.")
+
+(defconst org-timeline-height 0
+  "Computed height (number of lines) of the timeline.")
```

Comment:
If these are computed, I think they shouldn't be marked as constants, since they can change between invocations of agenda.  Only really unchangable things should be marked as constants, the reason is that also the byte compiler can possibly in-line them and produce incorrect code (that is, instead of the variable it will just place the value there which won't change if in the future the variable is re-assigned).

Reply:
Well, I read that "defconst are the least constant variables in elisp". `defvar` doesn't seem to work, and from what I read `defconst` was the way to go for variables that weren't customizable.

Reply:
Here is the [link](http://endlessparentheses.com/what-s-a-defconst-and-why-you-should-use-it.html) to where I read that.

Although it seems like it wasn't the reason I ran into problems. I'll change those to `defvar` again.